.storybook/preview-body.html

```diff
@@ -0,0 +1,16 @@
+<style>
```

Comment:
I found that I needed this in order for the button not to be unusually tall. In particular, the `box-sizing` and `margin` values.

Reply:
We might want to push these values into the default styling for button though?

Reply:
Yeah. Alternatively, we could add some global reset/normalize style to components.